### DISCUSSION TITLE
Ubo logger

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#2049] [Q&A]
**Firefox android/desktop stable 98.0** , 

ubo1.41.8
Ubo logger doesn't show these filters

```
vidshar.org##+js(acis, JSON.parse, break;case $.)
vidbam.org##+js(acis, JSON.parse, break;case $.)
```
![Screenshot_20220314-010744-691](https://user-images.githubusercontent.com/20338483/158076098-5ca6ba12-0e71-4978-bfce-2ce6c5ec32d8.png)

when surfed on `https://4hdmovie.com/movies/valimai-2022-tamil/`

**Comment by mapx-** [comment]
@gorhill
